add-ons/fabric8/fabric8.addon

```diff
@@ -0,0 +1,37 @@
+# Name: fabric8
+# Description: Installs fabric8 locally on Minishift
+# Required-Vars: GITHUB_OAUTH_CLIENT_ID,GITHUB_OAUTH_CLIENT_SECRET
```

Comment:
see here for how to pass them into the addon: https://github.com/minishift/minishift-addons/issues/4#issuecomment-320628096

instructions for creating these 2 env vars are here:
https://github.com/fabric8io/fabric8-platform/blob/master/INSTALL.md#setup-github-client-id-and-secret

one day we'd like to make these env vars optional; but for a little while they are gonna be mandatory